qase-pytest/setup.cfg

```diff
@@ -28,7 +28,7 @@ package_dir =
 setup_requires = pyscaffold>=3.2a0,<3.3a0
 # Add here dependencies of your project (semicolon/line-separated), e.g.
 install_requires =
-    qaseio>=2.2.4,<3.0
+    qaseio>=3.0.0a1
```

Comment:
Why not 3.0.0?

Reply:
historical reasons) it does not matter, because 3.0.0 > 3.0.0a1